tools/smartdnsblock/devdoc.md

```diff
@@ -0,0 +1,22 @@
+# `smartdnsblock`
```

Comment:
Let's rename it to README.md so it shows up on the Github UI.

Reply:
I did that for you :-)

Reply:
😄 I did that as well but forgot to push.

Reply:
Feel free to merge :-)